packages/block-editor/src/components/block-switcher/style.scss

```diff
@@ -123,7 +125,6 @@
 		margin: $grid-unit-20 0;
 		// Use padding to prevent the pattern previews focus style from being cut-off.
 		padding: 0 $grid-unit-20;
-		overflow: hidden;
```

Comment:
Looks like I may be undoing my own fix by removing this - #44079.

I think the root issue is that there's a sharing of classnames between two components that do very different things.

Reply:
Have resolved this by using a modifier classname `unset` to remove the overflow only for the patterns menu.

Unpicking the tech debt is unfortunately going to be time consuming, and might not be the best use of time.